pkg/testing/runner/runner.go

```diff
@@ -757,7 +781,12 @@ func (r *Runner) mergeResults(results map[string]OSRunnerResult) (Result, error)
 		if !ok {
 			return Result{}, fmt.Errorf("batch ID not found %s", id)
 		}
-		batchName := fmt.Sprintf("%s/%s/%s/%s", batch.LayoutOS.OS.Type, batch.LayoutOS.OS.Arch, batch.LayoutOS.OS.Distro, batch.LayoutOS.OS.Version)
+		var batchName string
+		if batch.LayoutOS.OS.Type == define.Linux {
+			batchName = fmt.Sprintf("%s/%s/%s/%s", batch.LayoutOS.OS.Type, batch.LayoutOS.OS.Arch, batch.LayoutOS.OS.Distro, batch.LayoutOS.OS.Version)
+		} else {
+			batchName = fmt.Sprintf("%s/%s/%s", batch.LayoutOS.OS.Type, batch.LayoutOS.OS.Arch, batch.LayoutOS.OS.Version)
+		}
```

Comment:
Nit: There are a few places with similar if-else statements for linux vs. not.  The only difference seems to be the separator (`-` or `/`) and, of course, the OS type (linux or not).  Maybe pull this out into a single helper function?